docs/wiki/99 - Developers/Release Testing Guide.md

```diff
@@ -0,0 +1,4 @@
+# Release Testing Guide
+
+
```

Comment:
```suggestion
* Browser: test on both Chrome and Firefox
* Help text: Does it render ok and does the content need updating? 

* Install-related? Test for both Ubuntu and CentOS - also check this version number against the Readme/s (which may need updating)
```

Reply:
I like the addition but I actually think that the install should have more space devoted to it. It should mention explicitly that an Ubuntu and CentOS install should be tested as well as a development install (i.e. to make sure we have all the dependencies up to date).

Reply:
yes, definitely 👍

Reply:
I'll apply your suggestion and we can iterate further.